frontend/packages/console-app/locales/ja/console-app.json

```diff
@@ -403,15 +409,14 @@
   "{{ cpuMessage }}": "{{ cpuMessage }}",
   "{{ memoryMessage }}": "{{ memoryMessage }}",
   "{{ machineHealthCheckLabelPlural }} automatically remediate node health issues.": "{{ machineHealthCheckLabelPlural }} はノードの正常性の問題を自動的に修正します。",
-  "Conditions": "条件",
+  "Conditions": "Conditions",
```

Comment:
was this a missed translation?

Reply:
> was this a missed translation?

I saw this change in more than one place, and I think it is intentional since the string "Conditions" was not newly marked. In the past, I have followed up with the Localization team on similar changes, and they confirmed it is intentional. I will double-check with them on this.